library/pickle.po

```diff
@@ -943,6 +1300,14 @@ msgid ""
 "protocol version is used as well as the number of items to append, so both "
 "must be supported.)"
 msgstr ""
+"Un itérateur (non pas une séquence). Les éléments qu'il fournit sont ajoutés "
+"à l'objet un par un avec sa méthode :meth:`append`, ou bien plusieurs à la "
+"fois avec sa méthode :meth:`extend`. Ceci est principalement utile aux les "
```

Comment:
```suggestion
"à l'objet un par un avec la méthode :meth:`append`, ou bien plusieurs à la "
"fois avec la méthode :meth:`extend`. Ceci est principalement utile aux les "
```
Je pense qu'il s'agit des méthode de l'objet à désérialiser et non de l'itérateur.